build.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -e
+
+# --- Script Variables ---
+ARTIFACT_NAME="oracle-toolkit-infra-manager.jar"
```

Comment:
If someone runs this script twice, what behavior do we want?  Should we have uniquely-named files?  Or explicitly remove existing files?

Also, I should note that this is a zip rather than a jar;  it doesn't contain the jar manifests.